File: week02/day04/src/SimpleReplace.java
/**
 * Created by Bálint on 2017. 03. 23..
 */
public class SimpleReplace {
  public static void main(String... args){
    String example = "In a dishwasher far far away";
    String replacedExample = example.replace("dishwasher", "galaxy");


    // I would like to replace "dishwasher" with "galaxy" in this example, but it has a problem.
    // Please fix it for me!
    // Expected ouput: In a galaxy far far away

    System.out.println(replacedExample);
  }
}

File: javascript_practice/3-data-structures/simplereplace.js
'use strict';

var example = "In a dishwasher far far away";

// I would like to replace "dishwasher" with "galaxy" in this example
// Please fix it for me!
// Expected ouput: In a galaxy far far away
var wrongWord = 'dishwasher';
var correction = 'galaxy';

if (example.indexOf(wrongWord) !== -1) {
  example = example.replace(wrongWord, correction);
}

console.log(example);
